reproschema/models/reproschema

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
is this file needed in this repo?

Reply:
I thought that first we will release `reproschema` and after this I will point to the specific version